presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/glue/TestHiveGlueMetastore.java

```diff
@@ -96,4 +112,29 @@ public void testStorePartitionWithStatistics()
     {
         testStorePartitionWithStatistics(STATISTICS_PARTITIONED_TABLE_COLUMNS, BASIC_STATISTICS_1, BASIC_STATISTICS_2, BASIC_STATISTICS_1, EMPTY_TABLE_STATISTICS);
     }
+
+    @Test
+    public void testGetPartitions()
+    {
+        try {
+            createDummyPartitionedTable(tablePartitionFormat, CREATE_TABLE_COLUMNS_PARTITIONED);
+            Optional<List<String>> partitionNames = getMetastoreClient().getPartitionNames(tablePartitionFormat.getSchemaName(), tablePartitionFormat.getTableName());
+            assertTrue(partitionNames.isPresent());
+            assertEquals(partitionNames.get(), Arrays.asList("ds=2016-01-01", "ds=2016-01-02"));
+        }
+        catch (Exception ex) {
```

Comment:
This catch block is not needed -- any exception thrown by the test method will fail the test.

Reply:
Sure. Had to add `throws` to the parent class method too.